paf-mvp-core-js/src/crypto/verifier.ts

```diff
@@ -30,7 +36,10 @@ export class Verifier<T> {
       const publicKey = await this.publicKeyProvider(signingDomain);
       const signature = this.definition.getSignature(signedData);
       const toVerify = this.definition.getInputString(signedData);
-      result.isValid = publicKey.verify(toVerify, signature);
+
+      const verifier = this.builder.buildVerifier(publicKey);
```

Comment:
FAR It would be interesting to cache the verifier including the error for a given PEM.